src/TuDa.CIMS.Api/Program.cs

```diff
@@ -1,7 +1,14 @@
+using Microsoft.EntityFrameworkCore;
 using TuDa.CIMS.Api;
+using TuDa.CIMS.Api.Database;
```

Comment:
⛏️ Remove unused imports
```suggestion
using TuDa.CIMS.Api;
```